### ISSUE TITLE
[BUG] Unable to use a migrated account to log in through Microsoft. 

### ISSUE THREAD

**Issue by TheoCGaming** [PojavLauncherTeam/PojavLauncher#1955]
### Describe the bug

I can't log in through Microsoft on PojavLauncher. Microsoft asks "Do you want PojavLauncher to access... blah blah blah." As soon as I tap "Yes" the browser window immediately closes and I'm taken back to PojavLauncher without it logging me in. There isn't even an error message.

### Steps To Reproduce

```markdown
1. Start PojavLauncher
2. Press "MICROSOFT LOGIN"
3. Attempt to log in to PojavLauncher with a migrated account (if that works, try setting your default browser to Firefox and let me know if that doesn't work.)
```

### Expected Behavior

I expect my account to be logged in at the news screen. 

### Platform

```markdown
- Device model: Samsung Galaxy S9+
- CPU architecture: Qualcomm Snapdragon 845
- Android version: 10
- PojavLauncher version: Latest Play Store version as of 9/1/2021
```

### Anything else?

I'm using Firefox as my default browser.

**Comment by khanhduytran0** [comment]
You need to login through the "Microsoft login" button inside the launcher, not from web browser.

**Comment by TheoCGaming** [comment]
> You need to login through the "Microsoft login" button inside the launcher, not from web browser.

I've been doing that. It redirects me to a web browser (which would normally be Chrome)

**Comment by Mathias-Boulay** [comment]
Wait, can you provide us video footage of the issue ?  @TheoCGaming 
Btw, for the version go to options > about
In the launcher

**Comment by TheoCGaming** [comment]
I just tried again on a newer version since it just updated as soon as I opened it, but there was no way to get to the about screen. When I tried to log on it came up with an error for Microsoft redirecting to another page. It was "start.ui.xboxlive.com/AddChildToFamily". I believe it only showed that because I'm under 18 and not added to a family, since a previous account we were using since it wasn't being used got suspended probably because Microsoft thought it was hacked.
